includes/admin/class-wc-admin-help.php

```diff
@@ -42,19 +42,13 @@ public function add_tabs() {
 				'title'   => __( 'Help &amp; Support', 'woocommerce' ),
 				'content' =>
 					'<h2>' . __( 'Help &amp; Support', 'woocommerce' ) . '</h2>' .
-					'<p>' . sprintf(
-						/* translators: %s: Documentation URL */
-						__( 'Should you need help understanding, using, or extending WooCommerce, <a href="%s">please read our documentation</a>. You will find all kinds of resources including snippets, tutorials and much more.', 'woocommerce' ),
-						'https://docs.woocommerce.com/documentation/plugins/woocommerce/?utm_source=helptab&utm_medium=product&utm_content=docs&utm_campaign=woocommerceplugin'
-					) . '</p>' .
 					'<p>' . sprintf(
 						/* translators: %s: Forum URL */
-						__( 'For further assistance with WooCommerce core you can use the <a href="%1$s">community forum</a>. If you need help with premium extensions sold by WooCommerce, please <a href="%2$s">use our helpdesk</a>.', 'woocommerce' ),
-						'https://wordpress.org/support/plugin/woocommerce',
-						'https://woocommerce.com/my-account/tickets/?utm_source=helptab&utm_medium=product&utm_content=tickets&utm_campaign=woocommerceplugin'
+						__( 'For further assistance with Classic Commerce you can use the <a href="%1$s">ClassicPress community forum</a>.', 'woocommerce' ),
+						'https://forums.classicpress.net/'
 					) . '</p>' .
-					'<p>' . __( 'Before asking for help we recommend checking the system status page to identify any problems with your configuration.', 'woocommerce' ) . '</p>' .
-					'<p><a href="' . admin_url( 'admin.php?page=wc-status' ) . '" class="button button-primary">' . __( 'System status', 'woocommerce' ) . '</a> <a href="https://wordpress.org/support/plugin/woocommerce" class="button">' . __( 'Community forum', 'woocommerce' ) . '</a> <a href="https://woocommerce.com/my-account/tickets/?utm_source=helptab&utm_medium=product&utm_content=tickets&utm_campaign=woocommerceplugin" class="button">' . __( 'WooCommerce helpdesk', 'woocommerce' ) . '</a></p>',
+					'<p>' . __( 'Before asking for help we recommend using the system status page to identify any problems with your configuration. You should make a copy of this report to add to your support request.', 'woocommerce' ) . '</p>' .
+					'<p><a href="' . admin_url( 'admin.php?page=wc-status' ) . '" class="button button-primary">' . __( 'System status', 'woocommerce' ) . '</a> <a href="https://forums.classicpress.net/" class="button">' . __( 'Community forum', 'woocommerce' ) . '</a></p>',
```

Comment:
'<p>' . __( 'Before asking for help we recommend using the system status page to identify any problems with your configuration. You should make a copy of this report to add to your support request.', 'woocommerce' ) . '</p>' .
					'<p><a href="' . admin_url( 'admin.php?page=wc-status' ) . '" class="button button-primary">' . __( 'System status', 'woocommerce' ) . '</a> <a href="https://github.com/ClassicPress-research/classic-commerce/issues" class="button">' . __( 'Plugin Github Issues Section.', 'woocommerce' ) . '</a></p>',

Reply:
Wade has created an area on the ClassicPress forum to get general CC support. I have added this in as a link.

Reply:
I believe this is a more focal point. https://forums.classicpress.net/c/support/classic-commerce

Reply:
Good point. Will change that.